packages/ag-charts-community/src/chart/series/polar/pieSeries.test.ts

```diff
@@ -117,3 +107,42 @@ describe('Doughnut', () => {
         await compare();
     });
 });
+
+/* eslint-disable no-console */
+describe('Validation', () => {
```

Comment:
Nit: We should aim for one root `describe()` block for a module, with a name describing the module under test (either `pieSeries.ts` (module filename) or `PieSeries` (main module export) - as test reports for the repo will merge this with other `Validation` describe blocks which are also declared at the top level.